src/components/Game.js

```diff
@@ -24,18 +45,16 @@ const Game = () => {
       <p className="Game__format-example">
         { exampleFormat }
       </p>
+      <RecentSubmission submission = {currentFields} showRecent={showRecent} />
```

Comment:
Just a note that `submission` is a string in the prop-types and an array of objects here.